server/resources/migrations/42_add_transactions_app_id_idx.down.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS transactions_app_id_idx;
```

Comment:
I would leave out the `if exists` here because something has gone wrong if the index doesn't exist when this migration is run.